sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -248,36 +248,13 @@ export class ProgramInfoRequestController extends BaseController {
         throw new BadRequestException("Application not found.");
       }
 
-      let studyStartDate = payload.studyStartDate;
-      let studyEndDate = payload.studyEndDate;
-      let selectedOfferingIntensity = payload.offeringIntensity;
-
-      let offeringToCompletePIR: EducationProgramOffering;
-      if (payload.selectedOffering) {
-        // Check if the offering belongs to the location.
-        const offeringLocation =
-          await this.offeringService.getOfferingLocationId(
-            payload.selectedOffering,
-          );
-
-        if (offeringLocation?.institutionLocation.id !== locationId) {
-          throw new UnauthorizedException(
-            "The location does not have access to the offering.",
-          );
-        }
-        studyStartDate = offeringLocation.studyStartDate;
-        studyEndDate = offeringLocation.studyEndDate;
-        selectedOfferingIntensity = offeringLocation.offeringIntensity;
-        // Offering exists, is valid and just need to be associated
-        // with the application to complete the PIR.
-        offeringToCompletePIR = {
-          id: payload.selectedOffering,
-        } as EducationProgramOffering;
-      }
+      const offeringToCompletePIR = {
+        id: payload.selectedOffering,
+      } as EducationProgramOffering;
 
       this.applicationService.checkOfferingIntensityMismatch(
         application.data.howWillYouBeAttendingTheProgram,
-        selectedOfferingIntensity,
+        payload.offeringIntensity,
```

Comment:
`payload.offeringIntensity` is part of the removed pir form, I guess. can you recheck, if we need this validation check